Bookish.Web/Controllers/HomeController.cs

```diff
@@ -47,6 +47,37 @@ public IActionResult Copies(string isbn)
             return View(model);
         }
 
+        public IActionResult NewBook(string title = "", string author = "", string isbn = "", int totalCopies = 1)
+        {
+            var message = bookishService.GetIsbnExists(isbn)
```

Comment:
I think the view model should be responsible for constructing any error message. You just need to tell the view model that the isbn exists, e.g. with a boolean. Also an error message should be named `errorMessage` rather than `message` 🙂